io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -545,6 +563,13 @@ impl<'a, 'b: 'a> AcceptFsm for &'a mut AcceptLink<'b> {
             .await
             .map_err(|e| (e, Some(close::reason::GENERIC)))?;
 
+        // Extension Patch
+        #[cfg(feature = "transport_compression")]
```

Comment:
See my comment on the need of the `ext_patch` in the `Open` message.
Besides that, the `cfg` directive is for compression and not for the patch.